src/test/java/core/basesyntax/CreateReportServiceTest.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax;
+
+import static core.basesyntax.db.FruitStorage.storage;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertThrows;

```

Comment:
A lot of useful things was added in 4.13 Junit, but in 4.12 JUnit we don`t have static import, and that a reason why we need use 4.12 in this task